src/RequestFlags.cc

```diff
@@ -23,3 +26,11 @@ RequestFlags::cloneAdaptationImmune() const
     return *this;
 }
 
+void
+RequestFlags::disableCacheUse(const char * const reason)
+{
+    debugs(16, 3, "for " << reason);
+    missCachingDecision = false; // may already be set to false
+    noCache = true; // may already be true
```

Comment:
Setting these flags to these values is always valid and previous state does not matter.
Only code setting the inverse values away from these needs the attention a comment brings.
```suggestion
    missCachingDecision = false;
    noCache = true;
```